helm-chart/renku/requirements.yaml

```diff
@@ -36,6 +36,6 @@ dependencies:
   version: 0.0.3
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
 - name: redis
-  version: 10.7.11
-  repository: "https://raw.githubusercontent.com/bitnami/charts/eb5f9a9513d987b519f0ecd732e7031241c50328/bitnami"
+  version: 17.4.2
+  repository: "https://charts.bitnami.com/bitnami"
```

Comment:
the reason for the explicit link before was that the bitnami repo broke, right? Should we not continue to follow the same pattern?

Reply:
so was the issue that the version we had was simply too old to be found somehow?

Reply:
Yes - iirc, bitnami had too many charts and it was taking ages to download indexes (indices?) - they decided to remove older charts from the index at `charts.bitnami.com` index and this caused problems in our reconciliations as the specified chart version could not be found in the `repository` - do I have that right @wesjdj ?

I think they articulated a policy of a 6 month window, but I might have that wrong...

Reply:
You are correct @seanrmurphy, 6 month retention of charts in the Bitnami repo index.

Reply:
What'd be ideal is if we pointed to a specific tarball for the chart version, rather than the repo and the version.